conformance/results/pytype/aliases_implicit.toml

```diff
@@ -43,8 +335,8 @@ Line 117: Expected 1 errors
 Line 118: Expected 1 errors
 Line 119: Expected 1 errors
 Line 133: Expected 1 errors
-Line 54: Unexpected errors ['File "aliases_implicit.py", line 54, in <module>: Invalid type annotation \\'[str, str]\\'  [invalid-annotation]', 'File "aliases_implicit.py", line 54, in <module>: Invalid type annotation \\'Callable[Concatenate[int, P], R][[str, str], None]\\'  [invalid-annotation]']
-Line 68: Unexpected errors ['File "aliases_implicit.py", line 68, in good_type_aliases: Callable[Concatenate, Any] [assert-type]']
-Line 72: Unexpected errors ['File "aliases_implicit.py", line 72, in good_type_aliases: Callable[[Any], None] [assert-type]']
-Line 105: Unexpected errors ['File "aliases_implicit.py", line 105, in <module>: Invalid type annotation \\'[int, str]\\' for p2 [invalid-annotation]', 'File "aliases_implicit.py", line 105, in <module>: Invalid type annotation \\'True\\' for p10 [invalid-annotation]', 'File "aliases_implicit.py", line 105, in <module>: Invalid type annotation \\'<instance of List[Type[int]]>\\' for p4 [invalid-annotation]', 'File "aliases_implicit.py", line 105, in <module>: Invalid type annotation \\'3\\' for p9 [invalid-annotation]', 'File "aliases_implicit.py", line 105, in <module>: Invalid type annotation \\'1\\' for p11 [invalid-annotation]', 'File "aliases_implicit.py", line 105, in <module>: Invalid type annotation \\'((int, str),)\\' for p3 [invalid-annotation]', 'File "aliases_implicit.py", line 105, in <module>: Invalid type annotation "{\\'a\\': \\'b\\'}" for p5 [invalid-annotation]']
+Line 54: Unexpected errors ["aliases_implicit.py:54:9: \\x1b[1m\\x1b[31merror\\x1b[39m\\x1b[0m: in <module>: Invalid type annotation '[str, str]'  [invalid-annotation]", "aliases_implicit.py:54:9: \\x1b[1m\\x1b[31merror\\x1b[39m\\x1b[0m: in <module>: Invalid type annotation 'Callable[Concatenate[int, P], R][[str, str], None]'  [invalid-annotation]"]
```

Comment:
Does this coloring show up correctly in the output tables we generate? It might be better to configure pytype to not use color, if possible.

Reply:
No, this doesn't show up in the output report. 

Unfortunately, I don't see a way to disable color. If there is, it's not documented.